supertokens/supertokens.go

```diff
@@ -54,6 +55,9 @@ func supertokensInit(config TypeInput) error {
 		superTokens.OnSuperTokensAPIError = config.OnSuperTokensAPIError
 	}
 
+	superTokens.Debug = config.Debug
+	superTokensInstance = superTokens
```

Comment:
why is this here?

Reply:
Because `superTokensInstance` is initialised only at the end of `supertokensInit()`, but its needed in the `logDebugMessage()` function.